client/src/features/Event/New/__test__/NewEventPage.test.tsx

```
@@ -1,96 +1,171 @@
+  MaxCapacityModal: ({ isOpen }: { isOpen: boolean }) =>
+    isOpen ? <div data-testid="capacity-modal">최대 수용 인원</div> : null,
+}));
 
-    expect(await screen.findByText(/템플릿 불러오기/)).toBeInTheDocument();
-  });
+vi.mock('../components/QuestionForm', () => ({
+  QuestionForm: () => (
+    <div>
+      <div>사전 질문</div>
+      <button>질문 추가</button>
+      <div>참가자에게 묻고 싶은 질문을 추가해 보세요.</div>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)